compiler/src/dotty/tools/backend/jvm/BCodeSkelBuilder.scala

```diff
@@ -629,6 +641,26 @@ trait BCodeSkelBuilder extends BCodeHelpers {
 
     } // end of method initJMethod
 
+    private def genTraitConstructorDefDef(dd: DefDef): Unit =
+      val statifiedDef = makeStatifiedDefDef(dd)
+      genDefDef(statifiedDef)
+
+    private def makeStatifiedDefDef(dd: DefDef): DefDef =
```

Comment:
"statified" is an interesting neologism :). Maybe add a comment explaining what it does to keep things clear?